src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -162,7 +162,8 @@ export class FairHeader extends React.Component<Props, State> {
     } = this.props
     const { width: screenWidth } = Dimensions.get("window")
     const imageHeight = 567
-    const { isSavedFairStateUpdating } = this.state
+    // const { isSavedFairStateUpdating } = this.state
+    // FIXME: When saving fairs is defined we can use commented code
```

Comment:
I'd prefer if we removed the `isSavedFairStateUpdating` state value and `handleSaveFair`. If it's not getting called, it should be removed – but we can still access it in git history.

We should add a reference to this pull request in the ticket to re-add the fair saving, though, to make it easy to add back in 👍

Reply:
ok sounds good